src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Figure {
+    void draw();
+
+    double getArea();
+}

```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The `draw()` and `getArea()` methods are not conceptually close to each other. Consider separating them into two different interfaces, such as `Drawable` for `draw()` and `AreaCalculable` for `getArea()`.